src/app/modules/calendar/calendar-month/calendar-month.component.ts

```diff
@@ -0,0 +1,51 @@
+import {Component, OnInit} from '@angular/core';
+import {NgClass, NgForOf, NgIf} from "@angular/common";
+import {CalendarService} from "../../../services/CalendarService";
+import moment from "moment";
+import {CalendarItem} from "../../../models/CalendarItem";
+import {MatFormFieldModule} from "@angular/material/form-field";
+import {MatInputModule} from "@angular/material/input";
+import {MatSelectModule} from "@angular/material/select";
+import {MatToolbar, MatToolbarRow} from "@angular/material/toolbar";
+import {MatIcon} from "@angular/material/icon";
+import {MatButton, MatIconButton} from "@angular/material/button";
```

Comment:
would be awesome to use some pattern for imports

for example:
1) 3rd party libs imports (Angular, material etc)
2) internal imports

please check how to setup eslint for automatization such a case and add a plugin that can help with imports sorting